backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -221,3 +207,47 @@ class EntityColumns:
         DbDatasetArtifact: dataset_asset_cols,
         DbDatasetProcessingStatus: dataset_processing_status_cols,
     }
+
+
+def list_collections_curation(
```

Comment:
Why the move? Any actual changes?

Reply:
Shoot. This would have been an accident. Stupid merge conflicts. Let me see if I can revert this portion so that the commit history is accurate.

Reply:
Oh, ok, even though this got moved, it still retains you as the author. So we're good. No actual changes.